pom.xml

```diff
@@ -120,6 +120,7 @@
     <google-cloud-dataflow-java-proto-library-all.version>0.5.160304</google-cloud-dataflow-java-proto-library-all.version>
     <guava.version>20.0</guava.version>
     <grpc.version>1.0.1</grpc.version>
+    <grpc-google-common-protos.version>0.1.0</grpc-google-common-protos.version>
```

Comment:
Yes, this is already coming from `Datastore` [here](https://search.maven.org/remotecontent?filepath=com/google/cloud/datastore/datastore-v1-protos/1.3.0/datastore-v1-protos-1.3.0.pom) and also used in other core `google` libraries like [here](https://github.com/GoogleCloudPlatform/google-cloud-java/blob/56da2c5359192b3ed4c8173d257c71b96c1bfdae/google-cloud-core/pom.xml#L125) we are just making it explicit to keep the dependency plugin happy.